### PR TITLE
New readme section on how to use permissions for CRUD controllers

### PR DESCRIPTION
**Pull request by Polfo**
I spent some time finding out how to integrate Permissions with CRUD controllers. It is now documented and could be useful for other users.
Here are some related questions without answer:
https://stackoverflow.com/questions/65469302/laravel-backpack-rolewise-permission-give-to-module
https://stackoverflow.com/questions/57982478/laravel-backpackcrud-set-permission-role-to-a-category
https://stackoverflow.com/questions/55489878/cannot-restrict-access-to-permission-manager-routes-using-role-based-middleware
Did someone publish an alternative elsewhere? 
Can you accept this proposal? There are many ways to do it, but this may help getting started. It is the kind of documentation I missed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,7 +8,7 @@
 [![Style CI](https://styleci.io/repos/58740020/shield)](https://styleci.io/repos/58740020)
 [![Total Downloads][ico-downloads]][link-downloads]
 
-Admin interface for [spatie/laravel-permission](https://github.com/spatie/laravel-permission). It allows admins to easily add/edit/remove users, roles and permissions, using [Laravel Backpack](https://laravelbackpack.com). 
+Admin interface for [spatie/laravel-permission](https://github.com/spatie/laravel-permission). It allows admins to easily add/edit/remove users, roles and permissions, using [Laravel Backpack](https://laravelbackpack.com).
```
Comment (Polfo) [inline README.md:11]: My editor always removes trailing spaces on save.

File: README.md [hunk 2]
```diff
@@ -227,6 +227,164 @@ This package also adds Blade directives to verify whether the currently logged i
 
 You can use Laravels native @can directive to check if a user has a certain permission.
 
+## Use permissions in CRUD controllers
+
+CRUD controllers have methods to [dynamically allow or deny access](https://backpackforlaravel.com/docs/4.1/crud-api#access) to operations. The ```$this->crud->allowAccess()``` and ```$this->crud->denyAccess()``` methods control both:
```
Comment (tabacitu) [inline README.md:232]:
```suggestion
CRUD controllers have methods to [dynamically allow or deny access](https://backpackforlaravel.com/docs/6.x/crud-api#access) to operations. The ```$this->crud->allowAccess()``` and ```$this->crud->denyAccess()``` methods control both:
```

File: README.md [hunk 3]
```diff
@@ -227,6 +227,164 @@ This package also adds Blade directives to verify whether the currently logged i
...
+- the form's navigation buttons display like Add, Edit, Delete and
+- the security access guards, returning a 403 forbidden error when no permission.
+
+Here is a way to link the permissions to CRUD controller access.
+Reminder: permissions can be assigned to a user either directly or through a role.
```
Comment (tabacitu) [inline README.md:237]:
```suggestion
In most cases, you should use `access` and `permission` separately. But if you want to link them together, so a "permission" gives "access", here is how you can do that too. Reminder: permissions can be assigned to a user either directly or through a role.
```

**Review by tabacitu — CHANGES_REQUESTED** [review]

**Comment by tabacitu** [comment]
Wow, this is VERY GOOD. Thanks a lot @Polfo , I've just given this a read and merged it, super-useful. Thanks again @Polfo and sorry for taking so long to merge, must have slipped through the cracks.

Cheers!

**Pull request merged**